dll/win32/setupapi/install.c

```diff
@@ -697,7 +693,12 @@ static BOOL register_dlls_callback( HINF hinf, PCWSTR field, void *arg )
         if (!SetupGetIntField( &context, 4, &flags )) flags = 0;
 
         /* get timeout */
+#ifdef __REACTOS__
+        /* "11,,cmd.exe,,,/K dir" means default timeout, not a timeout of zero */
+        if (!SetupGetIntField( &context, 5, &timeout ) || timeout == 0) timeout = 60;
```

Comment:
```suggestion
        if (!SetupGetIntField(&context, 5, &timeout) || timeout == 0) timeout = 60;
```
I performed this style-suggestion for the case that an upstream patch will be sent to Wine.

Reply:
Why? The style used in that file is spaces around the parameters and I was just following that.

Reply:
Yes and no. Wine is typically not consistent in their formatting style. And at the same time they accept style improvements only for lines that have to be touched anyway for improving functionality. Therefore it might be a chance to improve the style here. That's why I suggested to include it as a proposal for the upstream fix.